adapter_hci.go

```diff
@@ -50,11 +50,24 @@ func (a *hciAdapter) enable() error {
 }
 
 func (a *hciAdapter) Address() (MACAddress, error) {
+	if a.hci.address.MAC != [6]uint8{} {
```

Comment:
```suggestion
	if a.hci.address != MACAddress{} {
```
Probably, easier to understand and hopefully is better future-proof?

Reply:
Very good idea, I will do that.

Reply:
Done.

Reply:
@ysoldak the reqquired syntax was slightly different, but the idea behind your request about readability and "futureproofness" was the important part. :+1: